### ISSUE TITLE
Plugin is checking all the mirrors

### ISSUE THREAD

**Issue by stdevel** [stdevel/check_pakfire#14]
The plugin currently checks all the mirrors instead of picking the first working one:

```
$ ./check_pakfire.py
ERROR:check_pakfire:Unable to validate mirror 'https://ftp.gwdg.de/pub/linux/ipfire/pakfire2/2.21': '<urlopen error [SSL: SSLV3_ALERT_HANDSHAKE_FAILURE] sslv3 alert handshake failure (_ssl.c:661)>'
ERROR:check_pakfire:Unable to validate mirror 'https://mirror1.ipfire.org/pakfire2/2.21': '<urlopen error [Errno -2] Name or service not known>'
OK: Core update (125) up2date, packages up2date
```

As a result, the plugin takes quite long to check the system state and Nagios/Icinga might return a timeout:
```
$ time ./check_pakfire.py
...
real    0m36.631s
user    0m10.301s
sys     0m0.163s
```

**Comment by stdevel** [comment]
Investigating in branch `bug-all_mirrors-issue14`

**Comment by stdevel** [comment]
Commit bd35c11e3056bcc65566da9fe00251f6723492a1 fixes this issue:
```
$ time ./check_pakfire.py
...
real    0m0.789s
user    0m0.441s
sys     0m0.027s
```

**Comment by stdevel** [comment]
Fixed in commit a39c5395986b615a668a4313582c46cb00137151.
